app/components/profile_picture/edit_image/index.tsx

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {Platform, View} from 'react-native';
+
+import {Client} from '@client/rest';
+import CompassIcon from '@components/compass_icon';
+import {useServerUrl} from '@context/server_url';
+import {useTheme} from '@context/theme';
+import useDidUpdate from '@hooks/did_update';
+import NetworkManager from '@init/network_manager';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import UserProfileImage from '../image';
+
+type ProfilePictureProps = {
+    canUpdateProfilePicture: boolean;
+    imageUri?: string;
+    isProfileImageRemoved?: boolean;
+    lastPictureUpdate: number;
+    size?: number;
+    testID?: string;
+    userId: string;
+};
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            justifyContent: 'center',
+            alignItems: 'center',
+            borderRadius: 80,
```

Comment:
why is this radius 80? doesn't this has direct relation with the size prop ?

Reply:
corrected